src/course-outline/header-navigations/HeaderNavigations.jsx

```diff
@@ -90,6 +92,7 @@ const HeaderNavigations = ({
           {intl.formatMessage(messages.viewLiveButton)}
         </Button>
       </OverlayTrigger>
+      <CourseOutlineAnalyticsSlot hasSections={hasSections} sections={sections} />
```

Comment:
Open questions:
* Do we want to allow site operators to customize this beyond adding things at the end? If so, then it would make sense to have this slot wrap the existing buttons instead.
* This slot is not analytics specific, any ideas for a more generic name for this slot?

Reply:
Replied [above](https://github.com/openedx/frontend-app-authoring/pull/1725#discussion_r1993013679)